Tema2/pizza_order.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Pizza Order</title>
+    <link rel="shortcut icon" type="img/png" href="img/turn.png" />
+    <link rel="Stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <form action="form_submited.html" target="_blank">
+      <fieldset>
+        <legend>Pizza order</legend>
+        <label for="nume">Your name:</label>
+        <input
+          type="text"
+          name="nume"
+          minlength="3"
+          maxlength="10"
+          placeholder="Your name"
+          required
+        />
+        <br /><br />
+
+        <label for="email">Your email:</label>
+        <input type="email" name="email" value="Add email" required />
+        <br /><br />
+        <label for="topping">Topping:</label>
```

Comment:
Regula este ca fiecare input sa aiba un label atasat. 
Aici era mai ok sa scrii asa:

```suggestion
        <p>Topping:</p>
        <input type="radio" id="supreme name="topping" value="Supreme" checked /><label for="supreme">Supreme</label>
        <input type="radio" id="vegetarian" name="topping" value="Vegetarian" /><label for="vegetarian">Vegetarian</label>
        <input type="radio" id="hawaian" name="topping" value="Hawaian" /><label for="hawaian">Hawaian</label>
```